buildpipeline/Core-Setup-RHEL7-x64.json

```diff
@@ -161,7 +161,7 @@
       "allowOverride": true
     },
     "BuildArguments": {
-      "value": "--skip-prereqs --configuration $(BuildConfiguration) --targets Default --env-vars NUGET_API_KEY=$(NUGET_API_KEY),GITHUB_PASSWORD=$(GITHUB_PASSWORD),REPO_PASS=$(REPO_PASS),CONNECTION_STRING=$(CONNECTION_STRING)",
+      "value": "--skip-prereqs --configuration $(BuildConfiguration) --targets Default --env-vars NUGET_API_KEY=$(NUGET_API_KEY),GITHUB_PASSWORD=$(GITHUB_PASSWORD),REPO_PASS=$(REPO_PASS),CONNECTION_STRING=\'$(CONNECTION_STRING)\'",
```

Comment:
I think the `\`s would actually cause an error because there's no need to escape the `'`s.

Reply:
Maybe `\\'`? Have you already tried just `'`?